src/dwarf/resolver.rs

```diff
@@ -46,12 +46,68 @@ impl From<&DWSymInfo<'_>> for SymInfo {
     }
 }
 
+#[derive(Clone, Debug)]
+enum Either<A, B> {
+    A(A),
+    B(B),
+}
```

Comment:
I tried to understand why we need Either<A, B>, but I'm failing to see that from the code, so must be some Rust reason. What am I missing?

Reply:
discussed offline, it's the type strictness thing, as I suspected